code/modules/clothing/outfits/factions/independent.dm

```diff
@@ -540,20 +540,25 @@
 	dcoat = /obj/item/clothing/suit/hooded/wintercoat/cargo
 	shoes = /obj/item/clothing/shoes/sneakers/brown
 	glasses = /obj/item/clothing/glasses/sunglasses
-	l_hand = /obj/item/clipboard
+	r_pocket = /obj/item/clipboard
```

Comment:
i'm not sure if clipboards fit in pockets

Reply:
They do.